From cd113449bd46ce07c7cfd1797414ed5eba4a674c Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Thu, 6 Jun 2019 11:25:00 +0900
Subject: [PATCH 08/11] shared/sec: refactor updating classifier info

Rename spp_classifier_mac_update() to update_classifier(), and revise
names of vars and log messages in the function for refactoring.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 .../secondary/spp_worker_th/cmd_utils.c       |  2 +-
 src/shared/secondary/spp_worker_th/vf_deps.h  |  2 +-
 src/vf/classifier_mac.c                       | 23 ++++++++++---------
 3 files changed, 14 insertions(+), 13 deletions(-)

diff --git a/src/shared/secondary/spp_worker_th/cmd_utils.c b/src/shared/secondary/spp_worker_th/cmd_utils.c
index ae6be1a..d5faa35 100644
--- a/src/shared/secondary/spp_worker_th/cmd_utils.c
+++ b/src/shared/secondary/spp_worker_th/cmd_utils.c
@@ -891,7 +891,7 @@ update_comp_info(void)
 
 #ifdef SPP_VF_MODULE
 		if (comp_info->wk_type == SPPWK_TYPE_CLS)
-			ret = spp_classifier_mac_update(comp_info);
+			ret = update_classifier(comp_info);
 		else
 			ret = spp_forward_update(comp_info);
 #endif /* SPP_VF_MODULE */
diff --git a/src/shared/secondary/spp_worker_th/vf_deps.h b/src/shared/secondary/spp_worker_th/vf_deps.h
index 9b59b3d..8d3cd81 100644
--- a/src/shared/secondary/spp_worker_th/vf_deps.h
+++ b/src/shared/secondary/spp_worker_th/vf_deps.h
@@ -60,7 +60,7 @@ free_mac_classifier(struct mac_classifier *mac_clf)
  * @retval SPP_RET_OK If succeeded.
  * @retval SPP_RET_NG If failed.
  */
-int spp_classifier_mac_update(struct sppwk_comp_info *comp_info);
+int update_classifier(struct sppwk_comp_info *comp_info);
 
 /**
  * Update forwarder info.
diff --git a/src/vf/classifier_mac.c b/src/vf/classifier_mac.c
index 232ed19..f6074e4 100644
--- a/src/vf/classifier_mac.c
+++ b/src/vf/classifier_mac.c
@@ -695,26 +695,27 @@ spp_classifier_mac_init(void)
 
 /* classifier(mac address) update component info. */
 int
-spp_classifier_mac_update(struct sppwk_comp_info *wk_comp_info)
+update_classifier(struct sppwk_comp_info *wk_comp_info)
 {
-	int ret = SPP_RET_NG;
-	int id = wk_comp_info->comp_id;
-	struct management_info *mng_info = g_mng_infos + id;
-	struct cls_comp_info *cmp_info = NULL;
+	int ret;
+	int wk_id = wk_comp_info->comp_id;
+	struct management_info *mng_info = g_mng_infos + wk_id;
+	struct cls_comp_info *cls_info = NULL;
 
 	RTE_LOG(INFO, SPP_CLASSIFIER_MAC,
-			"Component[%u] Start update component.\n", id);
+			"Start updating classifier, id=%u.\n", wk_id);
 
-	cmp_info = mng_info->cmp_infos + mng_info->upd_index;
+	/* TODO(yasufum) rename `infos`. */
+	cls_info = mng_info->cmp_infos + mng_info->upd_index;
 
 	/* initialize update side classifier information */
-	ret = init_component_info(cmp_info, wk_comp_info);
+	ret = init_component_info(cls_info, wk_comp_info);
 	if (unlikely(ret != SPP_RET_OK)) {
 		RTE_LOG(ERR, SPP_CLASSIFIER_MAC,
-				"Cannot update classifier mac. ret=%d\n", ret);
+				"Cannot update classifier, ret=%d.\n", ret);
 		return ret;
 	}
-	memcpy(cmp_info->name, wk_comp_info->name, STR_LEN_NAME);
+	memcpy(cls_info->name, wk_comp_info->name, STR_LEN_NAME);
 
 	/* change index of reference side */
 	mng_info->upd_index = mng_info->ref_index;
@@ -729,7 +730,7 @@ spp_classifier_mac_update(struct sppwk_comp_info *wk_comp_info)
 	uninit_component_info(mng_info->cmp_infos + mng_info->upd_index);
 
 	RTE_LOG(INFO, SPP_CLASSIFIER_MAC,
-			"Component[%u] Complete update component.\n", id);
+			"Done update classifier, id=%u.\n", wk_id);
 
 	return SPP_RET_OK;
 }
-- 
2.17.1

